File: ds.js

const array = [9,7,6,0,5,2,4];
const target = 10;
let new_ar =[];
for(let i=0; i<array.length; i++){
    for(let j=i+1; j<array.length; j++){
        if(array[i] + array[j] == target){
            new_ar.push(array[i], array[j]);
        }
    }
}
console.log(new_ar);

File: ds.java
public class sample {
    public static int[] twoNumberSum(int[] array, int target) {
        for ( i = 0; i<array.length-1 , i++) {
            for (j=i+1; j<array.length; j++) {
                if ( array[i] + array[j] == target) {
                    return new int [] {array[i], array[j]};
                }
            }
        }
        return new int [0];
    }
    public static void main(String[] args) {
        int[] array = {9,7,6,0,2,5,4};
        int target = 10;
        int[] result =  twoNumberSum(array, target);
        System.out.println( result );
    }
}
